feature detection example with get (#41725)
mdn/content: files/en-us/web/javascript/reference/functions/get/index.md

};
```

### Feature detection

> Many functions accept an object and retrieve individual properties from it as separate parameters (this object parameter is known as an _options bag_). You can detect whether a specific option is supported by detecting if the property has been retrieved with a getter. This example checks if the `colorType` option is supported on the {{domxref("HTMLCanvasElement.getContext()")}} method.

Reviewer: ```suggestion
Many functions accept an object and retrieve individual properties from it as separate parameters (this object parameter is known as an _options bag_). You can detect whether a specific option is supported by using a getter to track if the property has been retrieved. This example checks if the `colorType` option is supported by the {{domxref("HTMLCanvasElement.getContext()")}} method.
```
